config/config.hocon.sample

```diff
@@ -0,0 +1,27 @@
+###########################################
+#    Sample configuration for             #
+# gcp-dataflow-streaming-example-project  #
```

Comment:
I think it would be clearer to reference `# google-cloud-dataflow-example-project #` here